includes/file_helpers.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <cstdio>
+#include <memory>
+#include <string>
+#include <windows.h>
+
+static inline size_t file_get_size(FILE* f){
```

Comment:
Is `size_t` what you want here? I thought this datatype was supposed to be for array indices.

Reply:
size_t is just an unsigned integer type that is always big enough to fit the whole memory range of the program